tools/sentry-release.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
```

Comment:
Copied from Integreat.

Reply:
Minor: Could be refactored to a .ts file